app-frontend/src/app/pages/imports/datasources/detail/detail.html

```diff
@@ -81,6 +81,7 @@ <h1 class="h3">{{$ctrl.datasource.name}}</h1>
                 <th>Index</th>
                 <th>Number</th>
                 <th>Name</th>
+                <th>Wavelength</th>
```

Comment:
This isn't a problem you introduced, but we're missing some header cells which causes the table border to not render:

![image](https://user-images.githubusercontent.com/2442245/42385583-2e2ab0d0-810b-11e8-9a38-8a1984707c10.png)

We need a total of six cells to match the body.